src/Integration/MefServices/VsSessionHost.cs

```diff
@@ -81,7 +75,6 @@ internal sealed class VsSessionHost : IHost, IProgressStepRunnerWrapper, IDispos
                 credentialStoreService,
                 projectToLanguageMapper,
                 configurationProvider,
-                configFilePathProvider,
```

Comment:
The `configFilePathProvider` was only being imported because it was needed to construct the configuration persister -> removed.